app/src/main/res/layout/actiivty_notification.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Could you mimic the the existing slides? e.g https://github.com/syncthing/syncthing-android/blob/main/app/src/main/res/layout/activity_firststart_slide_storage.xml

Reply:
fixed

Reply:
Could you rename the file to match the others?

Reply:
`activity_firststart_slide_notification.xml` ?